{"raw":{},"required":["Filename","Size"],"package":"python3-pytest-instafail","source":"pytest-instafail","version":"0.5.0-1","section":"universe/python","priority":"optional","architecture":"all","depends":["python3-pytest","python3:any"],"suggests":["python-pytest-instafail-doc"],"installedSize":33,"maintainer":"Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>","description":"plugin for py.test that shows failures and errors instantly","homepage":"https://github.com/pytest-dev/pytest-instafail","filename":"pool/universe/p/pytest-instafail/python3-pytest-instafail_0.5.0-1_all.deb","size":6052,"md5":"3d442da4757ba847ea76f5fb018dd89c","sha1":"99d5acabe7f770b3e385695262c719ab0f3a7db4","sha256":"8780d9232ba18b50054f2c8b03985d67c77d89ad8ea8dd7a0d5529fba5f8ae8a","sha512":"115ad7b525e3c0f31283af0c456f57a55b0b2aca857e82200635c860316a936a7079c2caeb5f6dfc3e2ce4ff682932352c0e04709374123b3d7b1d343f4aa72c","descriptionMd5":"4acabc61b3293224ea4a27802dc75041"}